Makefile

```diff
@@ -164,7 +164,7 @@ build-dll: node_modules
 	@mkdir -p build
 	@CALYPSO_ENV=$(CALYPSO_ENV) $(NODE_BIN)/webpack --display-error-details --config webpack-dll.config.js
 
-build-server: install
+build-server: install server/devdocs/proptypes-index.json
```

Comment:
What environments will this actually run in? Do we actually need to generate a full proptypes index in the test environment?